dotCMS/src/main/java/com/dotmarketing/quartz/DotJobStore.java

```diff
@@ -34,7 +36,22 @@
 
 	@Override
 	public void storeJobAndTrigger(SchedulingContext ctxt, JobDetail newJob, Trigger newTrigger) throws ObjectAlreadyExistsException, JobPersistenceException {
-		super.storeJobAndTrigger(ctxt, newJob, newTrigger);
+
+        if (StatefulJob.class.isAssignableFrom(newJob.getJobClass())) {
+            this.executeInLock(this.isLockOnInsert() ? "TRIGGER_ACCESS" : null, conn -> {

```

Comment:
Codacy found an issue: [Assigning an Object to null is a code smell.  Consider refactoring.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5971105)